databind/src/main/java/gov/nist/secauto/metaschema/databind/model/metaschema/impl/FlagContainerSupport.java

```diff
@@ -106,11 +107,14 @@ public FlagContainerSupport(
       flagInstances.merge(flag.getEffectiveName(), flag, (v1, v2) -> {
         if (LOGGER.isErrorEnabled()) {
           IModelDefinition owningDefinition = v1.getContainingDefinition();
+          IModule module = owningDefinition.getContainingModule();
           LOGGER.error(
-              String.format("Unexpected duplicate flag instance name '%s' in definition '%s' in module '%s'",
+              String.format(
+                  "Unexpected duplicate flag instance name '%s' in definition '%s' in module name '%s' at '%s'",
                   key,
                   owningDefinition.getName(),
-                  owningDefinition.getContainingModule().getShortName()));
+                  module.getShortName(),
+                  module.getLocation()));
```

Comment:
Non-blocking: even my silly mistakes in test cases bring real value, one for the history books!